src/libraries/Directory.Build.targets

```diff
@@ -170,6 +170,7 @@
   <!-- Adds Nullable annotation attributes to C# non .NETCoreApp builds. -->
   <ItemGroup Condition="'$(Nullable)' != '' and
                         '$(Nullable)' != 'disable' and
+                        '$(NoIncludeNullableAttributes)' != 'true' and
```

Comment:
>NoIncludeNullableAttributes [](http://example.com/codeflow?start=27&length=27)

nit: 

Should we call it `SkipAddingNullableAttributes`? 

CC @ViktorHofer

Reply:
Renamed to `SkipIncludeNullableAttributes`.